<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RMTSI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="StartWidget" name="tabStart">
       <attribute name="title">
        <string>Start</string>
       </attribute>
      </widget>
      <widget class="SetupWidget" name="tabSetup">
       <attribute name="title">
        <string>Setup</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabResults">
       <attribute name="title">
        <string>Results</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>697</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SetupWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.setupWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StartWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui.startWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
